src/api/cpp/cvc5.h

```diff
@@ -4281,6 +4281,9 @@ class CVC5_EXPORT Solver
   /**
    * Declare a symbolic pool of terms with the given initial value.
    *
+   * For details on how pools are used to specify instructions for quantifier
+   * instantiation, see documentation for the INST_POOL kind.
```

Comment:
```suggestion
   * instantiation, see documentation for the #INST_POOL kind.
```
Can you check if this correctly links to the `INST_POOL` kind with this fix?